frontend/src/components/App.tsx

```diff
@@ -671,3 +674,10 @@ function Subtitle({ children }: SubtitleProps) {
         </Typography>
     );
 }
+
+function oneMonthFromNow() {
+    const date = new Date();
+    const thisMonth = date.getMonth();
+    date.setMonth(thisMonth + 1);
```

Comment:
TIL that setting the month on a date object uses the modulus of whatever number you use. That's super fun. Everybody loves whimsy.

Reply:
💀